### ISSUE TITLE
log trail is missing gamma.dev

### ISSUE THREAD

**Issue by newtewt** [medic/cht-core#6770]
**Describe the issue**
Logtrail is used to view our logs for the different instances.  When looking at the dev instance of log trail it is not showing gamma.dev or gamma-b.dev

**Describe the improvement you'd like**
Find out why it is missing our instance and get it added. 

**Describe alternatives you've considered**
No alts available.

**Comment by nomulex** [comment]
I restarted the fleuntd daemonset on the node that hosts the gamma pod  and the logs started streaming in.  I think ES has closed the connection for this fluentd daemonset. 
![image](https://user-images.githubusercontent.com/3070004/101326684-af1ed400-387e-11eb-8820-0cee00441436.png)

**Comment by nomulex** [comment]
Moving this back to ready as fluentd on this pod seems to be failing.

**Comment by henokgetachew** [comment]
From my investigation, the number of restarts for gamma and gamma-b is suspiciously high. 
I also see this error repeatedly in the fluentd pod. 

`+0000 [error]: #0 Cannot allocate memory @ io_getpartial`

So it appears the error is that memory couldn't be allocated. The instance size is not small and everything on this instance is dev/test related. So, it doesn't make sense that there are lots of restarts and that memory is being eaten up. I also don't think it makes sense to give it a bigger instance because these are test instances that should not in theory need large size servers.

I suspected that memory was taken by the various medic-api instances in each of the pods and restarted every single one. This hasn't seemed to help.

**Comment by nomulex** [comment]
So I dug into this and I discovered that when we were deploying the k8s daemon set, we set the memory limit to 200Mi. Now the fluentd flush buffer on the node labeled testing was hiting the memory limit and K8s was actively killing all fluent threads with a SIGKILL `Worker 0 finished unexpectedly with signal SIGKILL` . This meant that fluent was not pushing logs to elastic.  I have grown the memory limit to 400Mi , also reduced the size of the flush buffer and deployed new fluentd daemon sets across all nodes in our cluster.  I have monitored them and I see that  k8s is no longer actively killing workers. I will keep monitoring  for about 24 hours to see if we still have this pod memory pressure issue.
